packages/rocketchat-mongo-config/server/index.js

```diff
@@ -0,0 +1,21 @@
+import { Mongo } from 'meteor/mongo';
+
+import tls from 'tls';
+// FIX For TLS error see more here https://github.com/RocketChat/Rocket.Chat/issues/9316
+// TODO: Remove after NodeJS fix it, more information https://github.com/nodejs/node/issues/16196 https://github.com/nodejs/node/pull/16853
+tls.DEFAULT_ECDH_CURVE = 'auto';
```

Comment:
do we still need this? seems node have fixed it already.

this is not related to mongo btw, so I don't it should be in this package =) but my vote is to remove.

Reply:
Done